Add route for t06 .../comments.

lanc2112b opened the pull request:
Add tests for path, all tests pass.
Add commentController.js.
Add commentModel.js.
FIX Missing semi colon in articleModel

lanc2112b committed:
Add route for t06 .../comments.
Add tests for path, all tests pass.
Add commentController.js.
Add commentModel.js.
FIX Missing semi colon in articleModel

Beth-McCormack commented on models/commentModel.js:
```diff
@@ -0,0 +1,20 @@
+        ORDER BY created_at DESC;`,
+      [id]
+    )
+    .then((result) => {
```
Have a think about why the query would be returning empty rows and how we would handle this in different ways. 
a) article does not exist 
b) article exists but has no comments - would this be an error? What would you send back instead?

Beth-McCormack commented on __tests__/app.test.js:
```diff
@@ -60,6 +61,37 @@ describe("GET Endpoints", () => {
+    // Has: comment_id, votes, created_at, author, body, article_id. 
+    it("200: Returns object with array containing all comments belonging to a specific article id", () => {
+      return request(app)
+        .get("/api/articles/3/comments") // currently 2 comments
```
What if the user entered api/articles/banana? With your current code, would this return the correct error code.

Beth-McCormack reviewed (changes requested):
see comments

lanc2112b committed:
Update commentController to check article exists.
Update commentController to return 200 if article valid and no comments.
Update test for invalid article ID.
